### PR TITLE
Add an example of how to add secure edge terminated routes.

### PR DESCRIPTION
**Pull request by ramr**
Adds an example of how to route secure routes (edge-terminate) - aka something to showcase non-wildcard DNS secured routes.
@adellape / @bfallonf   PTAL thx.

### CODE REVIEW

File: install_config/install/deploy_router.adoc [hunk 1]
```diff
@@ -282,6 +282,74 @@ To remove a passphrase from a keyfile, you can run:
...
 
+Here is an example of how to use a secure edge terminated route with TLS
+termination occuring on the router before traffic is proxied to the
+destination. The secure edge terminated route specifies the TLS certificate
+and key information. The TLS certificate is served by the router front end.
+
+First, start up a router instance:
+
+----
+# odm router --replicas=1 --credentials=$KUBECONFIG --service-account=router
```
Comment (bmeng) [inline install_config/install/deploy_router.adoc]: typo, should be oadm

Comment (ramr) [inline install_config/install/deploy_router.adoc]: Aargh, my bad.   @bmeng good catch.

**Comment by bfallonf** [comment]
@ramr This looks ok to me. We can give it a quick clean after it merges.  @adellape ?

**Pull request merged**